commands/new_function.go

```diff
@@ -217,37 +223,32 @@ functions:
 
 func printAvailableTemplates(availableTemplates []string) string {
 	var result string
-	sort.Sort(StrSort(availableTemplates))
+	sort.Slice(availableTemplates, func(i, j int) bool {
+		return availableTemplates[i] < availableTemplates[j]
+	})
 	for _, template := range availableTemplates {
 		result += fmt.Sprintf("- %s\n", template)
 	}
 	return result
 }
 
-func duplicateFunctionName(functionName string, appendFile string) error {
+func duplicateFunctionName(functionName string, appendFile string) (*stack.Services, error) {
```

Comment:
Save for follow-up PR, but I think the read should happen outside the function `duplicateFunctionName` - we shouldn't want methods doing more than one thing.